specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -74,16 +68,104 @@
       "post": {
         "summary": "Detect anomaly status of the latest point in time series.",
         "description": "This operation generates a model using points before the latest one. With this method, only historical points are used to determine whether the target point is an anomaly. The latest point detecting operation matches the scenario of real-time monitoring of business metrics.",
-        "operationId": "LastDetect",
-        "parameters": [
+        "operationId": "Detection_LastDetect",
+        "parameters": [{
+          "name": "body",
+          "in": "body",
+          "description": "Time series points and period if needed. Advanced model parameters can also be set in the request.",
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/Request"
+          }
+        }],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/LastDetectResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Detect anomaly status of the latest point in time series example": {
+            "$ref": "./examples/LastDetect.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeseriesGroupId}/inconsistency/detect": {
+      "post": {
+        "summary": "Detect inconsistent time series from a group of similiar time series",
+        "description": " This operation helps detect the inconsistent series among a group series with similiar trend.",
+        "operationId": "DetectInconsistency",
```

Comment:
We recommend `{noun}_{verb}` pattern for operation IDs. It looks better in generated SDKs.